pkg/controllers/openshift_controller_manager/controller.go

```diff
@@ -71,8 +73,5 @@ func Setup(cfg *cpoperator.ControlPlaneOperatorConfig) error {
 		c.Run(ctx, 1)
 		return nil
 	}))
-	configInformers.Config().V1().Images().Informer().AddEventHandler(c.EventHandler())
```

Comment:
I'm guessing this just isn't needed "boilerplate" code anymore to get the informer to process events? Just want to make sure this wouldn't lead to watches not firing on those resources since we didn't add a hander (this is also just a question for my general knowledge no need to change).

Reply:
Yes, the change is on purpose. In 4.6, the controller no longer responds to informer events, but rather does a once every second sync:
https://github.com/openshift/library-go/blob/c803a7b7bb2c756c18edbe45af00ca00e6e911cd/pkg/operator/configobserver/config_observer_controller.go#L112